external/WICED/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config WL_BCM4390X
+	bool "BCM4390X Wireless Module support"
+	select NET_NOINTS
+	select WICED
+	select BOARD_INITTHREAD
+	select BCM4390X_WIRELESS
+		default y
```

Comment:
1. ```default``` and ```help``` have one more depth. Please align them.
2. Usaully config lists a type, a default, dependancies and help. But this config has dependancies before default. Could you re-list them for readability? This is not important to operate but makes developer happy.

Reply:
it will be changed as your guidance.
BR, HJ Lee.